packages/wp-source/src/libraries/__tests__/api.tests.ts

```diff
@@ -1,12 +1,15 @@
+import * as frontity from "frontity";
 import Api from "../api";
-import { fetch } from "frontity";
+import { Response as NodeResponse } from "node-fetch";
 
 jest.mock("frontity");
-const mockedFetch = (fetch as any) as jest.Mock;
+const mockedFrontity = frontity as jest.Mocked<typeof frontity>;
```

Comment:
I had to change how we mock the `frontity` module, because it was not correct before.